packages/notebook-extension/src/index.ts

```diff
@@ -126,6 +128,40 @@ const checkpoints: JupyterFrontEndPlugin<void> = {
   },
 };
 
+/**
+ * Add a command to close the browser tab when clicking on "Close and Shut Down"
+ */
+const closeTab: JupyterFrontEndPlugin<void> = {
+  id: '@jupyter-notebook/notebook-extension:close-tab',
+  autoStart: true,
+  requires: [IMainMenu],
+  optional: [ITranslator],
+  activate: (
+    app: JupyterFrontEnd,
+    menu: IMainMenu,
+    translator: ITranslator | null
+  ) => {
+    const { commands } = app;
+    translator = translator ?? nullTranslator;
+    const trans = translator.load('notebook');
+
+    const id = 'notebook:close-and-halt';
+    commands.addCommand(id, {
+      label: trans.__('Close and Halt'),
```

Comment:
```suggestion
      label: trans.__('Close Notebook and Shutdown Kernel'),
```

Reply:
If going with this suggestion then we'll likely need to use `Shut Down` instead of `Shutdown` for consistency with the other menu entries.